Vagrantfile

```diff
@@ -24,11 +25,48 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.hostmanager.manage_host = true
     config.hostmanager.ignore_private_ip = false
     config.hostmanager.include_offline = true
+
+    # Get the dynamic hostname from the running box so we know what to put in 
+    # /etc/hosts even though we don't specify a static private ip address
```

Comment:
I assume you found this somewhere on StackOverflow or the like. Can you link it?